src/components/videocard/VideoCard.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-// import { songimg1 } //from '../../asset'
 import './videocard.css'
 
-export const VideoCard = ({thumbnail, title,chanel_pic  }) => {
+export const VideoCard = ({ _id,thumbnail, title,description,creator,views,channel_pic,categoryName  }) => {
   return (
     <>
-    <Link to={{}}>
+    <Link to={`/play/${_id}`}>
     <div className='video-card'> 
       <img src= { thumbnail } className='video-card-img' />
       <div className='video-card-title-container'>
-      <img src={chanel_pic} className='channel-logo'/>
+      <img src={channel_pic} className='channel-logo'/>
```

Comment:
if possible add alt="" tag in the inside image.

Reply:
thanks santosh , good pick . okay I will do it